bindings/wasm/src/account/wasm_method_relationship.rs

```diff
@@ -0,0 +1,30 @@
+// Copyright 2020-2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use identity::did::MethodRelationship;
+use serde_repr::Deserialize_repr;
+use serde_repr::Serialize_repr;
+use wasm_bindgen::prelude::*;
+
+#[wasm_bindgen (js_name = MethodRelationship)]
+#[derive(Clone, Copy, Serialize_repr, Deserialize_repr, PartialEq, Debug)]
+#[repr(u8)]
```

Comment:
Do we still need `Clone`, `Copy`, `PartialEq`, `Debug` on this? Are they required trait bounds from `Serialize_repr`, and `Deserialize_repr`? If not can we remove them?

Reply:
I removed them.